api/applicationStagesApi.ts

```diff
@@ -0,0 +1,34 @@
+import { ApiPromise, EmptyPayload } from '../types/apiResponse';
+import { ApplicationStageData, ApplicationStagePatchData, ApplicationStagePostData } from '../types/applicationStage';
+import api from './api';
+import { APPLICATIONS_API_ENDPOINT } from './applicationsApi';
+
+export const APPLICATION_STAGES_API_ENDPOINT = 'stages';
```

Comment:
Would it make more sense if we define a function `makeApplicationStagesApiEndpoint(applicationId: number)` instead? From what I see in the below functions we are never using this constant as it is, and we always have to do ``${APPLICATIONS_API_ENDPOINT}/${applicationId}/${APPLICATION_STAGES_API_ENDPOINT}``, so it would be good to extract that out.